foodsaving/groups/tasks.py

```diff
@@ -15,3 +22,38 @@ def record_stats():
         points.extend(stats.get_group_stores_stats(group))
 
     write_points(points)
+
+
+def send_inactive_in_group_notification_to_user(user, group):
+    print('Sending email to userid=', user.id, 'regarding inactivity in groupid=', group.id)
+    email = prepare_user_inactive_in_group_email(user, group)
+    email.send()
+
+
+def send_removal_from_group_notification_to_user(user, group):
+    print('Sending email to userid=', user.id, 'regarding removal from groupid=', group.id)
+    email = prepare_user_removed_from_group_email(user, group)
+    email.send()
+
+        
+@db_periodic_task(crontab(hour='2',minute='0'))
+def process_inactive_users ():
+    now = timezone.now()
+
+    print("Processing inactive users")
```

Comment:
I'd remove those print statements, I guess they will clutter the log files otherwise.